backend/app/serializers/book_serializer.rb

```diff
@@ -31,4 +31,8 @@ class BookSerializer
   attribute :reading_status do |book, params|
     book.reading_lists.find_by(user_id: params[:current_user]&.id)&.status
   end
+
+  attribute :reading_list_id do |book, params|
+    book.reading_lists.find_by(user_id: params[:current_user]&.id)&.id
+  end
```

Comment:
Needed the `reading_list.id` here in order for me to update the status and/or delete it. 

FE will send the request to BE `/api/v1/reading_lists/:id` 

Lemme know a better implementation for this 👍

Reply:
I'd personally do the same so this is fine with me.